packages/wp-now/src/start-server.ts

```diff
@@ -0,0 +1,99 @@
+import WPNow, { WPNowOptions } from './wp-now';
+import { HTTPMethod } from '@php-wasm/universal';
+import express from 'express';
+import fileUpload from 'express-fileupload';
+import { portFinder } from './port-finder';
+
+function generateMultipartFormDataString(json, boundary) {
+	let multipartData = '';
+	const eol = '\r\n';
+
+	for (const key in json) {
+		multipartData += `--${boundary}${eol}`;
+		multipartData += `Content-Disposition: form-data; name="${key}"${eol}${eol}`;
+		multipartData += `${json[key]}${eol}`;
+	}
+
+	multipartData += `--${boundary}--${eol}`;
+	return multipartData;
+}
```

Comment:
it still seems like we could rename this function. it's _very_ awkward to make sense of the function name against what it's doing, because it's doing more than just generating a multi-part FormData string.

either that or rework what it's doing so that it matches better.

why not try and bring some symmetry with `requestBodyToString` and call it `requestBodyToMultipartFormData`?

Reply:
looking into it more it seems like `express` might be interpreting these requests for us and parsing them into JSON? maybe we need to `reconstructOriginalMultipartFormDataBody` _or_ find a way to fast-track these through `express` without the transformation

just thinking out loud here

Reply:
I renamed the function here https://github.com/WordPress/wordpress-playground/pull/188/commits/847a9388c356685ed5c61f38f92b0061d9260339. Thanks for the suggestion.

Yes, I would like to use express to proxy the requests _automatically_. Maybe the express logic should live in a different common package to be reused by different applications.

Reply:
Great conversation!